woocommerce/woocommerce
# Add Colombian locales

### All Submissions:


### Changes proposed in this Pull Request:

Add Colombia locales (states).

### How to test the changes in this Pull Request:

1. Switch to this PR branch.
2. Verify state list is available on Checkout page after selecting Colombia as shipping destination Country.

### Other information:


### Changelog entry

i18n: Added Colombian states to WooCommerce localization settings.
